docs/architecture/modernize-desktop/whats-new-with-netcore3- for-desktop.md

```diff
@@ -0,0 +1,147 @@
+# What&#39;s new with .NET Core 3.0 for Desktop?
+
+## From .NET Framework to .NET Core. The motivation behind .NET **Core**
+
+Since its inception in 2002, .NET Framework has evolved through the years to support many technologies like Windows Desktop, ASP.NET, Entity Framework, Windows Store and many others. All of them are very different in nature. Therefore, Microsoft was approaching this evolution by taking parts of the .NET Framework and creating a different application stack for each technology. That way, development capabilities could be customized for the needs of the specific stack, which maximized the potential of every platform. Of course, that lead to fragmentation on the versions of the .NET Framework maintained by different independent teams. All these stacks have a common structure, containing an App Model, a Framework and a Runtime, but they differ in the implementation of each of these parts.
```

Comment:
I think `Windows Forms` is the correct name.

```suggestion
Since its inception in 2002, .NET Framework has evolved through the years to support many technologies like Windows Forms, ASP.NET, Entity Framework, Windows Store and many others. All of them are very different in nature. Therefore, Microsoft was approaching this evolution by taking parts of the .NET Framework and creating a different application stack for each technology. That way, development capabilities could be customized for the needs of the specific stack, which maximized the potential of every platform. Of course, that lead to fragmentation on the versions of the .NET Framework maintained by different independent teams. All these stacks have a common structure, containing an App Model, a Framework and a Runtime, but they differ in the implementation of each of these parts.
```